src/runtime-rs/crates/resource/src/manager.rs

```diff
@@ -4,22 +4,26 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-use crate::network::NetworkConfig;
-use crate::resource_persist::ResourceState;
-use crate::{manager_inner::ResourceManagerInner, rootfs::Rootfs, volume::Volume, ResourceConfig};
-use agent::types::Device;
-use agent::{Agent, Storage};
 use anyhow::Result;
 use async_trait::async_trait;
+
```

Comment:
suggest clean up the crates dependency here. 
rules as above

Reply:
Thanks, fixed.